src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -267,45 +268,47 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter {
 
         protected virtual bool RequestIsCacheable(ActionExecutingContext filterContext) {
 
+            var itemDescriptor = string.Format("{0} (Controller: {1}, Action: {2})", filterContext.RequestContext.HttpContext.Request.RawUrl, filterContext.ActionDescriptor.ControllerDescriptor.ControllerName, filterContext.ActionDescriptor.ActionName);

```

Comment:
Ah sorry, misread. I'd add Area and the other data you mentioned except tenant (that's already logged).